src/modules/src/esp_deck_flasher.c

```diff
@@ -49,6 +49,105 @@ bool espDeckFlasherCheckVersionAndBoot()
   return true;
 }
 
+static uint32_t sequenceNumber;
+static uint32_t numberOfDataPackets;
+static uint8_t sendBuffer[ESP_MTU + 10 + 16];
```

Comment:
What are these `+10` and `+16` ? Would we benefit as having them as defines? Or are they obvious?

Reply:
Definitely not obvious! Added definitions to clarify these refer to overhead lengths.